index.html

```diff
@@ -218,6 +237,400 @@ <h2>
         </li>
       </ol>
     </section>
+    <section>
+      <h2>
+        <code><dfn>tab_strip</dfn></code> member
+      </h2>
+      <p>
+        The `tab_strip` member of the <a data-cite=
+        "appmanifest#web-application-manifest">Web Application Manifest</a> is
+        an <a data-cite="appmanifest#dfn-object">object</a> that contains
+        information about how the application is intended to behave in the
+        [=display mode/tabbed=] display mode. It has the following members:
+      </p>
+      <ul>
+        <li>[=tab_strip/home_tab=]
+        </li>
+        <li>[=tab_strip/new_tab_button=]
+        </li>
+      </ul>
+      <section>
+        <h3>
+          <code><dfn>home_tab</dfn></code> member
+        </h3>
+        <p>
+          The `home_tab` member of the [=tab_strip=] object is an ordered map
+          that contains information about a special "home tab" that is intended
+          to serve as the top-level menu for the application. It contains the
+          following members:
+        </p>
+        <ul>
+          <li>[=home_tab/scope_patterns=]
+          </li>
+        </ul>
+        <p>
+          The <code><dfn>scope_patterns</dfn></code> member is a list of
+          {{URLPatternInput}}s that define the [=within home tab scope|scope of
+          the home tab=] relative to the [=manifest URL=].
+        </p>
+        <aside class="note">
+          Each scope pattern is either a string which looks like a URL but has
+          "<code>*</code>" as a wildcard character, or a dictionary containing
+          separate URL components with "<code>*</code>" wildcards. See
+          [[urlpattern]] for more information.
+        </aside>
+        <p>
+          An application <dfn>has a home tab</dfn> if the [=Document/processed
+          manifest=] includes a non-null [=home_tab=] member of the
+          [=tab_strip=] member.
```

Comment:
Should the `has a home tab` definition also  say the display mode needs to be tabbed? Or is it implied that none of these fields are relevant in that case?

Reply:
Hmm, yes, I think you're right. It's a little tricky since technically applications don't have display modes, application contexts do. So it's a little wrong to say "if the application's display mode is X" but I think we just need to go with that since that clearly shows the intention.

We do need something like this because later we say "whenever a new application window is created ... the user agent MUST create a new home tab context" which clearly can't be true for non-tabbed apps.